README.md

```diff
@@ -29,11 +54,64 @@ restart the portal: `make restart service=portal`
 pull images for all services: `make pull`
 
 pull images for portal: `make pull service=portal`
-
-
-
-
-
-
-
-_por necesidad o aventura, todo descubrimiento nace con el inicio de un viaje sin fin, el camino hacia la frontera_
\ No newline at end of file
+<br />
```

Comment:
Don't need this for markdown

Reply:
I tried adding two spaces and got weird behavior, esp adding multiple newlines, i'll try it again. Do you know of any other way to add newlines?